package.json

```diff
@@ -3,12 +3,19 @@
   "version": "0.23.3",
   "description": "Connexions (cnx.org) web site",
   "scripts": {
+    "precommit": "lint-staged",
     "prepublish": "bower install --config.analytics=false",
     "upgrade": "npm update; bower update --config.analytics=false",
     "test": "grunt test --verbose",
     "posttest": "node ./script/check-templates-for-non-i18n-text.js",
     "dist": "grunt dist --verbose"
   },
+  "lint-staged": {
+    "*.less": [
+      "stylelint --syntax=less --custom-formatter=node_modules/stylelint-formatter-pretty",
```

Comment:
Could/should this also include the `--fix` arg?

Reply:
Yes, this will autofix some of the issues according to the ruleset.